### PR TITLE
refactor nav js so only one list opens at a time

### PR DESCRIPTION
**Pull request by hippogriffic**
blah

@incuna/frontend pls
I'm sure this can be neater

### CODE REVIEW

File: scripts/style-guide.js [hunk 1]
```diff
@@ -24,7 +24,13 @@
         });
 
         $('.js-toggle-header').click(function () {
-            $(this).toggleClass('open').next('.js-toggle-body').toggleClass('open');
+            if ($(this).hasClass('open')) {
+                $(this).removeClass('open').next('.js-toggle-body').removeClass('open');
+            } else {
+                $('.js-toggle-header').removeClass('open');
```
Comment (Minglee01) [inline scripts/style-guide.js]: `$('.js-toggle-header', '.js-toggle-body').removeClass("open");`
maybe

File: scripts/style-guide.js [hunk 2]
```diff
@@ -24,7 +24,13 @@
...
+                $('.js-toggle-body').removeClass('open');
+                $(this).addClass('open').next('.js-toggle-body').addClass('open');
```
Comment (Minglee01) [inline scripts/style-guide.js]:
```
$('.js-toggle-header').click(function () {
            $(this).removeClass('open').next('.js-toggle-body').toggleClass('open');
            if (!$(this).hasClass('open')) {
               $('.js-toggle-header', '.js-toggle-body').removeClass("open");
            }
        });
```

File: scripts/style-guide.js (hunk 1 above)
Comment (hippogriffic) [reply scripts/style-guide.js]: doesn't seem to work :/

Comment (hippogriffic) [reply scripts/style-guide.js]: ah, has to be `$('.js-toggle-header, .js-toggle-body')`

File: scripts/style-guide.js (hunk 2 above)
Comment (hippogriffic) [reply scripts/style-guide.js]: have to do the `if` bit first, but this worked, thanks :)

**Comment by Minglee01** [comment]
@hippogriffic  good to merge if happy

**Pull request merged**